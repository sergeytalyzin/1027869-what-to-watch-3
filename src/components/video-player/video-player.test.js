import React from "react";
import renderer from "react-test-renderer";

import VideoPlayer from "./video-player.jsx";

const mock = {
  isPlaying: true,
  videoSrc: `somePath`,
  posterSrc: `somePath`
};

it(`VideoPlayer is renderer correctly`, ()=>{
  const {isPlaying, videoSrc, posterSrc} = mock;

  const tree = renderer.create(<VideoPlayer
    isPlaying={isPlaying}
    src={videoSrc}
    poster={posterSrc}
    handleMouseEnter={()=>{}}
    handleMouseOut={()=>{}}
  />, {
    createNodeMock: () => {
      return {play() {}};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
