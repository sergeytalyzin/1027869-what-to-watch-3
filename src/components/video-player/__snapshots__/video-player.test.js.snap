// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer is renderer correctly 1`] = `
<div
  className="small-movie-card__image"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <video
    className="player__video"
    poster="somePath"
    src="somePath"
  />
</div>
`;
